app/models/maestrano/connector/rails/health_check.rb

```diff
@@ -0,0 +1,28 @@
+module Maestrano::Connector::Rails
+  class HealthCheck
+    # Check API connection with Connec!
+    # any code that returns blank on success and non blank string upon failure
+    def self.perform_connec_check
+      organization = Maestrano::Connector::Rails::Organization.first
+      if Maestrano::Connector::Rails::ConnecHelper.connec_version(organization)
+        ''
+      else
+        'Connec!'
+      end
+    rescue => e
+      "Connec!: #{e}. "
+    end
+
+    # Check API connection with Dev Platform!
+    # any code that returns blank on success and non blank string upon failure
+    def self.perform_dev_platform_check
+      if Maestrano.auto_configure
```

Comment:
I'm worried it'll re-configure the connector at every health check request.
Could we just find a way to query the DevPlatform without re-configuring the connector, or maybe just checking that the connector is configured enough? (by looking at the `Maestrano` config?)